code/harness/Tester.cpp

```diff
@@ -506,6 +506,11 @@ int Tester::test_check_random_permutations(const int num_rounds) {
         duration_cast<milliseconds>(snapshot_end_time - snapshot_start_time);
     // End snapshot timing.
 
+    if (verbose) {
+      std::cout << "Test " << rounds + 1 << ": Writing " << permutes.size()
```

Comment:
Test #Number (Meant adding a # before rounds+1, as we write to the log this way)